ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tensor.py

```diff
@@ -1465,6 +1465,88 @@ def test_tensorflow__xor__(
     )
 
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="tensorflow.constant",
+    method_name="floordiv",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+        large_abs_safety_factor=10,
+        small_abs_safety_factor=10,
+        safety_factor_scale="log",
+    ),
+)
+def test_tensorflow_floordiv_(
+    dtype_and_x,
+    frontend,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+    backend_fw,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        backend_to_test=backend_fw,
```

Comment:
Why are we adding two test functions here? Looks like the only difference in these is that in (1) you're calling `helpers.get_dtypes("numeric")` while in the second you're calling `helpers.get_dtypes("float")`. You should actually use `helpers.get_dtypes("valid") and write only one test.